### DISCUSSION TITLE
Using TinaCMS with Git LFS

### DISCUSSION THREAD

**Discussion by jevawin** [tinacms/tinacms#4293] [Help]
Has anyone used Tina CMS with Git LFS?

I've synced my media but everything shows as missing files in the media manager. However, I am able to upload a new image and that shows fine.

<img width="697" alt="image" src="https://github.com/tinacms/tinacms/assets/25133187/ae3539ba-fe9e-460b-95a5-3cb3db23057c">

Is it just that Tina doesn't support Git LFS and I should move the files into the repo normally? (Or I could use an external hosting system.)

**Comment by kldavis4** [comment]
Git LFS is not currently supported
